include/pagmo/algorithms/compass_search.hpp

```diff
@@ -68,8 +69,7 @@ namespace pagmo
  * a standard in the scientific computing community for exactly the reason observed
  * by Davidon: it is slow but sure'.
  *
- * @note This algorithm does not work for multi-objective problems, nor for
- * constrained optimization or stochastic problems.
+ * @note This algorithm does not work for multi-objective problems, nor for stochastic problems.
```

Comment:
Yeps, I was delaying the issue of the evolve documentations as its missing everywhere. I added the docs for compass search to this PR